.env

```diff
@@ -6,6 +6,7 @@ FRONTEND_DOMAIN=localhost:3000
 HASH_COLLAB_QUEUE_NAME=collab
 HASH_EMAIL_TRANSPORTER=dummy
 
+NEXT_PUBLIC_HASH_OPENSEARCH_ENABLED=false
```

Comment:
I understand why we use `NEXT_PUBLIC_*` here but I wonder if we can actually avoid it. If we don’t come up with some workaround, then every time we’ll need a new var in UI, we'll have to touch other packages. Seeing `NEXT_PUBLIC_* outside Next.js is a bit confusing.

Since we’ve [managed to ‘feed’ our frontend with the monorepo‘s `.env*` files](https://github.com/hashintel/hash/pull/185), how about doing this in `next.config.js`?

```diff
 const { buildStamp } = require("./buildstamp");

 config({ silent: true, path: "../../.." });
+process.env.NEXT_PUBLIC_X = process.env.X
+process.env.NEXT_PUBLIC_Y = process.env.Y

 const sentryWebpackPluginOptions = {
```

Top-level `.env*` files won’t have to be aware of Next!

Reply:
It is a bit confusing but then again having two different names for env variables would also be. I thought of just making them separate but I don't like the idea of the same having having two different variable names

Reply:
We don't have to use `NEXT_PUBLIC_*` vars in multiple places, so we won't have too many cases when a variable is referred under two names. We can add prefixes in `next.config.js` and then remove them, say, in `src/lib/public-env.ts`:

```ts
export const X = process.env.NEXT_PLUBLIC_X;
export const Y = process.env.NEXT_PLUBLIC_Y;
```

That’s not ideal, but probably a bit better than adding `NEXT_PUBLIC_*` outside `packages/hash/frontend`. If we agree to use `NEXT_PUBLIC_*` only in two files (`next.config.js` and `public-env.ts`), maybe it won’t be too bad.

Reply:
Doing it this way seems fine then, I've updated it to use this